.github/title-labeler.yml

```diff
@@ -0,0 +1,20 @@
+version: v1
+
+labels:
+  - label: 'bug'
```

Comment:
We've never once tagged a PR as "bug"; in my mind that's reserved for issues.

Reply:
I feel like some way to indicate that a PR is high priority (i.e. fixes a bug) is useful. Can we use a different tag? I.e. `bug-fix`?

Reply:
I made this change in a4bd8a93abb40e178ebcc001af8e3443898f1c82.